### PR TITLE
Multiple http requests in single command for storm docs (SYN-2435)

### PR DESCRIPTION
**Pull request by rakuy0**
Should still support any previously existing docs too.

### CODE REVIEW

File: synapse/tests/test_lib_rstorm.py [hunk 1]
```diff
@@ -197,6 +203,19 @@ async def test_lib_rstorm(self):
             self.isin('inet:ipv4=5.6.7.8', text)  # one mock at a time
             self.isin('it:dev:str=notjson', text)  # one mock at a time
 
+            # multi reqest in 1 rstorm command
+            path = s_common.genpath(dirn, 'http_multi.rst')
+            with s_common.genfile(path) as fd:
+                fd.write(multi_rst_in_http.encode())
+            text = await get_rst_text(path)
```
Comment (rakuy0) [inline synapse/tests/test_lib_rstorm.py:210]: ah. forgot to add the assertions

File: synapse/lib/rstorm.py [hunk 2]
```diff
@@ -74,6 +75,37 @@ async def _mockHttp(self, *args, **kwargs):
...
+            # in any of those cases, default to using vcr
+            try:
+                with open(path, 'r') as fd:
+                    byts = json.load(fd)
+            except (FileNotFoundError, json.decoder.JSONDecodeError):
+                byts = None
+
+            if not byts:
+                with vcr.use_cassette(os.path.abspath(path), **vcr_kwargs) as cass:
+                    yield cass
```
Comment (rakuy0) [inline synapse/lib/rstorm.py:97]: It is worth it even yield this? Maybe for a future version?

Comment (invisig0th) [reply synapse/lib/rstorm.py:97]: I dont see a need, but it's also harmless :D

File: synapse/lib/rstorm.py [hunk 3]
```diff
@@ -150,6 +181,8 @@ async def __anit__(self, rstfile):
             'storm-cortex': self._handleStormCortex,
             'storm-expect': self._handleStormExpect,
             'storm-mock-http': self._handleStormMockHttp,
+            'storm-vcr-opts': self._handleStormVcrOpts,
+            'storm-clear': self._handleStormClear,
```
Comment (invisig0th) [inline synapse/lib/rstorm.py:185]: lets tweak this to `storm-clear-http` just in case

**Review by invisig0th — CHANGES_REQUESTED** [review]

**Review by invisig0th — APPROVED** [review]

**Pull request merged**